src/ui/CMakeLists.txt

```diff
@@ -28,7 +28,15 @@ endmacro()
 get_build_info(python-site PYTHON_SITE)
 get_build_info(python-inc PYTHON_INCLUDE_DIR)
 get_build_info(sip-inc SIP_INCLUDE_DIR)
-get_build_info(pyqt4-sip PYQT4_SIP)
+
+list(APPEND CMAKE_MODULE_PATH "/usr/share/apps/cmake/modules/") # needed for e.g Arch Linux as new versions of CMake only search a version-specific module path. Should be no problem if the path does not exist.
```

Comment:
Please keep this comment on a separate line and wrap the comments.  The rest of this file is near 80 columns as its limit, so try to stick near that limit within reason.

While it "should be no problem" if the path does not exist, we can avoid having to guess while being a little more tidy by just checking up front.

```cmake
if (EXISTS "/usr/share/apps/cmake/modules")
    # Arch Linux ...
    list(APPEND ...)
endif()
```